exporter/elasticsearchexporter/README.md

```diff
@@ -25,10 +25,18 @@ This exporter supports sending OpenTelemetry logs to [Elasticsearch](https://www
   [index](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices.html)
   or [datastream](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html)
   name to publish events to. The default value is `logs-generic-default`
+- `logs_dynamic_index` (optional): 
+  takes resource or log record attribute named `logs_index_prefix` and `logs_index_suffix`
+  resulting dynamically prefixed / suffixed indexing based on `logs_index`. (priority: resource attribute > log record attribute)
+  - `enabled`(default=false): Enable/Disable dynamic index for log records
 - `traces_index`: The
   [index](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices.html)
   or [datastream](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html)
   name to publish traces to. The default value is `traces-generic-default`.
+- `traces_dynamic_index` (optional):
+  takes resource or span attribute named `traces_index_prefix` and `traces_index_suffix`
```

Comment:
Not sure about the attributes naming:
1. `traces_`, `metric_`, `logs_` prefix seems redundant, since it's already set on spans, metrics or logs.
2. If you look at https://github.com/open-telemetry/opentelemetry-specification, attributes are usually separated by dots, not underscores.
3. Do we intend to apply the attribute to other exporters? If not, we should probably add the prefix `elasticsearch.`.

So maybe `elasticsearch.index.prefix` and `elasticsearch.index.suffix`?

Please let me know WDTY

Reply:
@dmitryax that makes absolutely sense to adjust the naming convention to the one defined in spec and also to consolidate the attributes to each prefix and suffix for every telemetry.
i will change it accordingly.

Reply:
@dmitryax adjusted :)